### PR TITLE
KAA-1356 Cross-compilation guide for C++ SDK (Intel Edison) on Linux platform

### PR DESCRIPTION
**Pull request by mlysenko**

### CODE REVIEW

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md [hunk 1]
```diff
@@ -91,3 +94,76 @@ You can follow [the Linux guide]({{root_url}}Programming-guide/Using-Kaa-endpoin
...
+        cd boost_1_58_0/
+        ./bootstrap.sh
+    
+    Now we can build Boost libraries for Edison target.
+    
+        source /opt/poky-edison/1.6.1/environment-setup-core2-32-poky-linux
+        sed -r 's/(using\s+gcc)(\s+;)/\1: : i586-poky-linux-g++ : <compileflags>-m32 -march=core2 -mtune=core2 -msse3 -mfpmath=sse -mstackrealign -fno-omit-frame-pointer --sysroot=$SDKTARGETSYSROOT\2/g' -i project-config.jam
+        ./b2 install --prefix=$SDKTARGETSYSROOT/usr/local
+
+    Copy `$SDKTARGETSYSROOT/usr/local/lib/libboost_*` object files to Edison's `/usr/local/lib` directory (e.g. using SSH, SCP, etc.).
```
Comment (vadimol) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: Not only the boost must be copied. We also need botan and avro.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md [hunk 2]
```diff
@@ -35,16 +35,15 @@ The following third-party components must be installed on the Intel Edison board
 To install these libraries, proceed as follows:
 
 1. Connect to the board either though the serial terminal or the ssh client.
-Refer to [the official Edison getting started guide](https://software.intel.com/en-us/iot/library/edison-getting-started) for any help required.
+Refer to [the official Edison getting started guide](https://software.intel.com/en-us/intel-edison-board-getting-started-guide) for any help required.
 
-2. Install dependencies.
+2. Install coreutils.
```
Comment (vadimol) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: It looks like you did not rebase your patch on the latest develop.

Comment (vadimol) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Edison/index.md]: These changes should not be here.

**Comment by vadimol** [comment]
Please, rebase your branch on the latest develop

**Comment by rasendubi** [comment]
LGTM

**Pull request merged**